plenum/test/pool_transactions/helper.py

```diff
@@ -254,6 +262,137 @@ def addNewStewardAndNode(looper, creatorClient, creatorWallet, stewardName,
     return newSteward, newStewardWallet, newNode
 
 
+def sdk_add_new_steward_and_node(looper,
+                                 sdk_pool_handle,
+                                 sdk_wallet_steward,
+                                 new_steward_name,
+                                 new_node_name,
+                                 tdir,
+                                 tconf,
+                                 allPluginsPath=None,
+                                 autoStart=True,
+                                 nodeClass=TestNode,
+                                 transformNodeOpFunc=None,
+                                 do_post_node_creation: Callable = None):
+    new_steward_wallet, steward_did = sdk_add_new_nym(looper,
+                                                      sdk_pool_handle,
+                                                      sdk_wallet_steward,
+                                                      alias=new_steward_name,
+                                                      role='STEWARD')
+
+    newNode = sdk_add_new_node(
+        looper,
+        sdk_pool_handle,
+        new_steward_wallet,
+        new_node_name,
+        steward_did,
+        tdir,
+        tconf,
+        allPluginsPath,
+        autoStart=autoStart,
+        nodeClass=nodeClass,
+        transformOpFunc=transformNodeOpFunc,
+        do_post_node_creation=do_post_node_creation)
+    return new_steward_wallet, newNode
+
+
+def sdk_add_new_nym(looper, sdk_pool_handle, creators_wallet,
+                    alias=None, role=None, node_count=None,
+                    seed=None):
+    seed = seed or randomString(32)
+    wh, _ = creators_wallet
+
+    # filling nym request and getting steward did
+    # if role == None, we are adding client
+    nym_request, new_did = looper.loop.run_until_complete(
+        prepare_nym_request(creators_wallet, seed,
+                            alias, role))
+
+    # sending request using 'sdk_' functions
+    request_couple = sdk_sign_and_send_prepared_request(looper, creators_wallet,
+                                                        sdk_pool_handle, nym_request)
+
+    # waitng for replies
+    node_count = node_count or 7
+    total_timeout = sdk_eval_timeout(1, node_count)
```

Comment:
remove two lines, it is not necessary to calc time out every time, you should calc only int case if test has some timeout specific checks or dependencies. for one req default timeout should be ok

node_count = node_count or 7
total_timeout = sdk_eval_timeout(1, node_count)

Reply:
Done